src/TcoCore/src/TcoCore.Wpf/Colors/TcoColors.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+
+namespace TcoCore.Wpf
+{
+    /// <summary>
+    /// Base color pallete is inspired by https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Brush-Names
+    /// It's a faceade of MaterialDesigngXaml color system.
+    /// </summary>
+    public class TcoColors
+    {
+        public static Brush Primary => GetDynamicBrush("PrimaryHueLightBrush", fallBackBrush: BrushFromHex("#5a7785"));
+        public static Brush OnPrimary => GetDynamicBrush("PrimaryHueLightForegroundBrush", fallBackBrush: Brushes.White);
+
+        public static Brush Secondary => GetDynamicBrush("SecondaryHueLightBrush", fallBackBrush: BrushFromHex("#bc5052"));
+        public static Brush OnSecondary => GetDynamicBrush("SecondaryHueLightForegroundBrush", fallBackBrush: Brushes.White);
+
+        public static Brush Accent => GetDynamicBrush("SecondaryHueMidBrush", fallBackBrush: Brushes.OrangeRed);
+        public static Brush OnAccent => GetDynamicBrush("SecondaryHueLightForegroundBrush", fallBackBrush: Brushes.White);
+
+        public static Brush Alert => new SolidColorBrush(GetDynamicColor("YellowPrimary500", ColorFromHex("#ffeb3b")));
+        public static Brush OnAlert => Brushes.Black;
+
+        public static Brush Error => GetDynamicBrush("MaterialDesignValidationErrorBrush", fallBackBrush: BrushFromHex("#dd2c00"));
+        public static Brush OnError => OnPrimary;
```

Comment:
Would you mind a simple test of public members? 😄 
You will probably need to set-up the tests for STA

Reply:
sure